control/joystick_parsing.h

```diff
@@ -117,7 +117,7 @@ typedef struct
 {
 	float x;	// Longitudinal (pitch)
 	float y;	// Lateral 		(roll)
-	float z;	// Vertical 	(thrust)
+	float z;	// Certical 	(thrust)
```

Comment:
Fixed in https://github.com/lis-epfl/MAVRIC_Library/commit/4e9ae60fe750c68289e41264ac712c0e600908a3